handler.py

```diff
@@ -222,8 +257,8 @@ def _key_exists(s3, bucket, key):
             return obj['Size']
 
 
-def _send_notification(message, context, dryrun=False):
-    if dryrun:
+def _send_notification(message, context):
+    if is_dry_run():
```

Comment:
Noice, I like that you didn't need to pass the dry run variable everywhere, much better!

Reply:
Yup. single source of config.